content/en/blog/choosing-a-driver.md

```diff
@@ -0,0 +1,185 @@
+
+_Source_: [github.com/falcosecurity/pdig](https://github.com/falcosecurity/pdig)
+
+#### Pros
+
+ - Lightweight, safe, and process specific
+ - Runs only in userspace
+ - Enables Falco for use cases when a kernel module, and an eBPF probe is not viable
+ 
+#### Cons
+
+ - The dependency on `ptrace(2)` is slow. Period.
+ - Requires executing Falco with the `pdig` binary to "hack" the driver.
+ 
+#### Summary
+
+The `pdig` tool is the most unique of all the drivers, and enables functionality not otherwise possible.
+
+ - Kubernetes
+ - AWS ECS/Fargate 
+ - AWS EKS/Fargate
+ - Environments where kernel modules and eBPF is not an option
+ 
+--- 
+
+## Suggested Cloud Provider Implementations 
+
+| Solution             | Suggested Driver            | More Resources                                                                                         |
+|----------------------|-----------------------------|--------------------------------------------------------------------------------------------------------|
+| Baremetal Kubernetes | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| Kubeadm Kubernetes   | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| Kubernetes Kind      | Kernel Module               | [Kind Documentation](https://falco.org/docs/third-party/#kind)                                         |
+| Minikube             | Kernel Module               | [Minikube Documentation](https://falco.org/docs/third-party/#minikube)                                 |
+| AWS EKS              | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| Azure                | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| GKE                  | eBPF Probe                  | [Falco on GKE](https://falco.org/docs/third-party/#gke)                                                |
+| IBM Cloud            | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| OpenShift            | Kernel Module               | [Helm Chart](https://falco.org/docs/third-party/#helm)                                                 |
+| AWS ECS              | pdig                        | [pdig](https://github.com/falcosecurity/pdig) [falco-trace](https://github.com/kris-nova/falco-trace)  |
+| AWS EKS (Fargate)    | pdig                        | [pdig](https://github.com/falcosecurity/pdig) [falco-trace](https://github.com/kris-nova/falco-trace)  |
```

Comment:
Should we add here also [falco-inject](https://github.com/fntlnz/falco-inject) maybe?

Reply:
Updated